### DISCUSSION TITLE
Request for Account Verification as a University Teacher (still pending)

### DISCUSSION THREAD

**Discussion by kreeksec** [community/community#140333] [GitHub Education] [answered]
### Hi everyone,

“Hello GitHub Campus Experts Support Team,

I would like to request an account verification that I am an active teacher at the university where I currently teach. I need your help to verify my status as a teacher in order to take advantage of the various features and programs provided by GitHub for teachers.

Thank you for your help and cooperation. I really appreciate your support.

Best regards,

**Comment by Thevandieg** [comment]
##  Hello deb0con

Thank you for reaching out! To verify your status as an active teacher, you'll need to apply for GitHub's GitHub Teacher benefits. Here’s how you can request verification:

Go to the GitHub Education Page: Visit [[GitHub Education](https://education.github.com/teachers)] and click on "Get benefits" under the teacher section.

Submit Proof of Employment: During the application process, you will be asked to provide proof that you're an active teacher. This can include documents such as:

- Your school-issued email address
- A faculty ID card
- A letter from the institution confirming your teaching role
### Wait for Verification: Once you submit the necessary documents, GitHub will review your application and verify your status. This process can take a few days.

~After your application is approved, you’ll gain access to GitHub's benefits for teachers, including free access to GitHub Classroom, educational resources, and other features to support your teaching activities.

**Comment by kreeksec** [comment, marked as answer]
I have submit Proof Of University Identitations but still pending there, could give more information why still pending
